linkerd/docs/config.md

```diff
@@ -142,6 +142,20 @@ Announce to ZooKeeper using the serverset format.
 * *pathPrefix* -- (optional) the ZooKeeper path under which services should be registered. (default:
   /discovery)
 
+#### io.l5d.announcer.Etcd
+
+Announce to Etcd using TTLs.
+
+* *host* -- (optional) the etcd API host.  (default: localhost)
+* *port* -- (optional) the etcd API port.  (default: 2379)
+* *pathPrefix* -- (optional) the etcd path under which services should be registered. (default:
+  /discovery)
+* *ttlSecs* -- (optional) the TTL in seconds for announcements to live in etcd.  Announcements will
+  disappear from etcd after this amount of time if not refreshed.  (default: 2 minutes)
+* *refreshSecs* -- (optional) the frequency in seconds to refresh the TTL.  This value should be set
+  smaller than the TTL to ensure that the announcement does not disappear from etcd. (default: 1
+  minute)
```

Comment:
should we add a warning or error in the event that this is larger than the TTL? (ignore this if there already is one, but I couldn't spot it)